### ISSUE TITLE
Syscheck cannot find some Firewall policy registers

### ISSUE THREAD

**Issue by okynos** [wazuh/wazuh#11090]
As reported in https://github.com/wazuh/wazuh/issues/11019#issuecomment-980453570 we have found some problems inside Windows Wazuh agent logs related with Syscheck module.

```
wazuh-agent: ERROR: (6704): Couldn't get path of '[x32] HKEY_LOCAL_MACHINE\\System\\CurrentControlSet\\Services\\SharedAccess\\Parameters\\FirewallPolicy\\RestrictedServices\\Configurable\\System:bed48bf0-5e43-4a13-9ab0-968e4b3ec414'
wazuh-agent: ERROR: (6704): Couldn't get path of '[x32] HKEY_LOCAL_MACHINE\\System\\CurrentControlSet\\Services\\SharedAccess\\Parameters\\FirewallPolicy\\RestrictedServices\\Configurable\\System:fbfcc1a9-2ff6-4779-a59a-7f370d97910e'
wazuh-agent: ERROR: (6704): Couldn't get path of '[x32] HKEY_LOCAL_MACHINE\\System\\CurrentControlSet\\Services\\SharedAccess\\Parameters\\FirewallPolicy\\RestrictedServices\\Configurable\\System:{9EA4BB8E-A899-4E3D-B32E-E6111D6A19EE}'
```
The error was found in:
- [Syscheck only test](https://github.com/wazuh/wazuh/issues/11036)

**Comment by MarcelKemp** [comment]
This is an error that appears whenever the registry has been inserted in the manager's database, but when synchronizing with the agent, that registry has already been deleted, so it cannot find it and generates the error.

> These registries that are shown in the error log are related to firewall rules generated by Windows. Apparently, these rules are loaded with each user login and are removed after logging out, to avoid problems such as the one shown in the following link: https://www.phy2vir.com/windows-server-2016-2019-rds-server-black-screen-or-start-menu-not-working/

I have not been able to reproduce this problem in a local environment, however it is also found in previous releases on Jenkins. An example where it appears is in the following issue found in version `v4.2.0-RC12`: 
* https://github.com/wazuh/wazuh/issues/9489

And the list of stress tests where the error has been found in version `v4.3.0-RC1`, is the following:
* https://github.com/wazuh/wazuh/issues/10995
* https://github.com/wazuh/wazuh/issues/10990
* https://github.com/wazuh/wazuh/issues/10993
* https://github.com/wazuh/wazuh/issues/11036
* https://github.com/wazuh/wazuh/issues/11046
* https://github.com/wazuh/wazuh/issues/11070
* https://github.com/wazuh/wazuh/issues/11076
* https://github.com/wazuh/wazuh/issues/11064

In addition, related to error `(6704)`, there are also errors of temporary files created and eliminated when executing the FIM stress test, appearing log messages such as the following:
```
2021/11/25 21:11:07 wazuh-agent: ERROR: (6704): Couldn't get path of 'c:\tmp\syscheck_test\files\fimstress.2400383'
2021/11/25 21:11:07 wazuh-agent: ERROR: (6704): Couldn't get path of 'c:\tmp\syscheck_test\files\fimstress.2400384'
2021/11/25 21:11:07 wazuh-agent: ERROR: (6704): Couldn't get path of 'c:\tmp\syscheck_test\files\fimstress.2400385'
2021/11/25 21:11:07 wazuh-agent: ERROR: (6704): Couldn't get path of 'c:\tmp\syscheck_test\files\fimstress.2400386'
```

These errors also appear in previous releases and in the current release. For instance:
* v4.2.0-RC12: https://github.com/wazuh/wazuh/issues/9497
* v4.3.0-RC1: https://github.com/wazuh/wazuh/issues/11065

**Comment by chemamartinez** [comment]
Hi team,

This issue will be included in the epic #9103 since this error appears during the FIM synchronization that will be delegated in RSync since v4.5.0.

That way, the error won't appear anymore which makes sense since it was an expected error if the entry was deleted before running the synchronization.

We will take advantage of this issue to remove the definition of the error in the development branch.

**Comment by chemamartinez** [comment]
Done at https://github.com/wazuh/wazuh/pull/15869.
